### PR TITLE
Add translator for Dutch newspaper NRC

### PR DESCRIPTION
**Pull request by mastaal**
Here's a PR for a translator for the Dutch newspaper [NRC](https://www.nrc.nl/). It works both for recent articles, as for re-published articles from their [archive](https://www.nrc.nl/index/archief/).

It's my first translator, so any feedback is much appreciated :smile:

### CODE REVIEW

File: NRC.nl.js [hunk 1]
```diff
@@ -0,0 +1,157 @@
...
+	along with Zotero. If not, see <http://www.gnu.org/licenses/>.
+	
+	***** END LICENSE BLOCK *****
+*/
+
+function detectWeb(doc, url) {
+	if (url.includes('/nieuws/')) {
+		return "newspaperArticle";
+	}
+
```
Comment (adam3smith) [inline NRC.nl.js:42]: We'd really want multiples (search results, maybe section browser) included, too

File: NRC.nl.js [hunk 2]
```diff
@@ -0,0 +1,157 @@
...
+	return false;
+}
+
+function doWeb(doc, url) {
+	return scrape(doc, url)
+}
+
+function scrape(doc, url) {
+	let item = new Zotero.Item("newspaperArticle");
```
Comment (adam3smith) [inline NRC.nl.js:51]: You'll want to start with the embedded metadata translator, which gets you most of the below (title, language, date, description, rights, url, snapshot) already. Use the template available in the translator editor

File: NRC.nl.js [hunk 3]
```diff
@@ -0,0 +1,157 @@
...
+
+	item.title = doc.querySelector("meta[property='og:title']").content
+	item.language = "nl"
+	item.url = doc.querySelector("meta[property='og:url']").content
+	item.abstractNote = doc.querySelector("meta[property='og:description']").content
+	item.rights = doc.querySelector("meta[name='dcterms.rights']").content
+	item.date = doc.querySelector("meta[property='og:url']").content.match(/\d\d\d\d\/\d\d\/\d\d/)[0].replace(/\//g, "-");
+
```
Comment (adam3smith) [inline NRC.nl.js:59]: We'd want authors where they're available, e.g. Clara van de Wiel for https://www.nrc.nl/nieuws/2023/06/15/lot-van-natuurherstelwet-timmermans-blijft-erg-onzeker-na-belangrijke-stemming-a4167267

**Review by adam3smith — CHANGES_REQUESTED** [review]
Some change requests -- let me know if anything isn't clear

**Comment by mastaal** [comment]
> Some change requests -- let me know if anything isn't clear

Thanks. This is useful feedback that I can work with.

**Comment by mastaal** [comment]
@adam3smith I think 2fb7fdb resolves all your comments.

File: NRC.nl.js [hunk 4]
```diff
@@ -0,0 +1,229 @@
...
+	along with Zotero. If not, see <http://www.gnu.org/licenses/>.
+	
+	***** END LICENSE BLOCK *****
+*/
+
+function detectWeb(doc, url) {
+	if (url.includes('/nieuws/')) {
+		return "newspaperArticle";
+	}
+	else if (url.endsWith(".nl/") || url.endsWith(".nl") || url.includes("/index/") || url.includes("/search/")) {
```
Comment (adam3smith) [inline NRC.nl.js:42]:
```suggestion
	else if ((url.endsWith(".nl/") || url.endsWith(".nl") || url.includes("/index/") || url.includes("/search/")) && getSearchResults(doc, true)) {
```

The last bit prevents detecting multiples where we can't import

**Comment by adam3smith** [comment]
Running the CI now to see if there are other issues -- see the nit about multiple detection above. Everything else looks great

**Comment by adam3smith** [comment]
Don't worry about the failed test

**Comment by adam3smith** [comment]
Thanks!

**Pull request merged**